safir/src/safir/pydantic/_validators.py

```diff
@@ -69,24 +81,35 @@ class Info(BaseModel):
 def normalize_isodatetime(v: str | None) -> datetime | None:
     """Pydantic field validator for datetime fields in ISO format.
 
-    This field validator requires the ISO 8601 date and time format with ``Z``
-    as the time zone (``YYYY-MM-DDTHH:MM:SSZ``). This format is compatible
-    with Kubernetes and the ISO UWS standard and is the same format produced
-    by `safir.datetime.isodatetime`. It should be used when the ambiguous
-    formats supported by Pydantic by default (such as dates and times without
-    time zone information) shouldn't be allowed.
+    This field validator requires the ISO 8601 date and time format, either
+    using ``Z`` as the timezone (``YYYY-MM-DDTHH:MM:SSZ``) or without timezone
```

Comment:
`YYYY-MM-DD[THH:MM:SS[.mmm]][Z]` is the clearest description of the acceptable format to me, but that may be because I've been staring at this PR for 10 minutes :smile: